backend/ECommerce.Api/appsettings.json

```
@@ -1,10 +1,18 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=Persistence/Data/ecommerce.db"
+  },
   "Logging": {
     "LogLevel": {
       "Default": "Information",
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "JWT": {
+    "Key": "your-secret-key-here-minimum-32-characters",
```

Fico pensando se faz sentido mesmo termos à mostra essas questões mais sensíveis como JWT e a conexão do banco de dados. Por mais que seja algo para estudo e desenvolvimento, precisamos pensar também nesses cenários. Penso que vale a pena pesquisar e verificar se podemos utilizar um .env ou algo similar para esses tipos de variável. E, só depois, chamá-las aqui dentro. Algo como:
 
```suggestion
    "Key": env.JWT_KEY,
```